app/helpers/automated_tests_helper.rb

```diff
@@ -1,12 +1,50 @@
 # Helper methods for Testing Framework forms
 module AutomatedTestsHelper
+  # Create a repository for the test scripts and test support files
+  # if it does not exist
+  def create_test_repo(assignment)
+    # Create the automated test repository
+    unless File.exist?(MarkusConfigurator
+                           .markus_config_automated_tests_repository)
+      FileUtils.mkdir(MarkusConfigurator
+                          .markus_config_automated_tests_repository)
+    end
+
+    test_dir = File.join(MarkusConfigurator
+                             .markus_config_automated_tests_repository,
+                         assignment.short_identifier)
+    unless !(File.exist?(test_dir))
```

Comment:
`if` and `unless` are not equivalent, they are opposites. Instead of using `if !(condition)` use `unless condition`.